Detectors/FIT/FT0/workflow/src/ReconstructionSpec.cxx

```diff
@@ -55,10 +58,10 @@ void ReconstructionDPL::run(ProcessingContext& pc)
   if (mUseMC) {
     LOG(info) << "Ignoring MC info";
   }
-  // auto caliboffsets = pc.inputs().get<o2::ft0::FT0ChannelTimeCalibrationObject*>("ft0offsets");
-  //  mReco.SetChannelOffset(caliboffsets.get());
-  auto caliboffsets = mCCDBManager.get<o2::ft0::FT0ChannelTimeCalibrationObject>("FT0/Calibration/ChannelTimeOffset");
-  mReco.SetChannelOffset(caliboffsets);
+  auto caliboffsets = pc.inputs().get<o2::ft0::FT0ChannelTimeCalibrationObject*>("ft0offsets");
+  mReco.SetChannelOffset(caliboffsets.get());
```

Comment:
you don't need to get/set the calibration at every TF, instead, use the mechanism described at https://docs.google.com/presentation/d/1WPNixo57zAn_gM6vMCIBOHs-DZqcKAkLX0CwUgO_VYg/edit#slide=id.g112eef84963_0_3